### PR TITLE
Min max scaler util

### PR DESCRIPTION
**Pull request by ebamberg**
## Description ##

This is MinMaxScaler implementation similar to sklearn.preprocessing.MinMaxScaler.
wanted to share this with you, just in-case that such tooling is from interest to add to the djl repo.

I needed such function in some time-series-forecast-demo and I haven't seen a similar implementation in the repo.  So mainly matches functionality I needed.

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by ebamberg** [comment]
added a function to detach to MinMaxScaler from NDManager

**Review by frankfliu — APPROVED** [review]
